static/js/answers-experience.js

```diff
@@ -0,0 +1,36 @@
+import DeferredPromise from './deferred-promise';
+
+export default class AnswersExperience {
+  constructor (runtimeConfig) {
+    this.runtimeConfig = runtimeConfig;
+
+    this.AnswersInitializedPromise = new DeferredPromise((resolve, reject) => {
+      setTimeout(reject, 5000, 
```

Comment:
Do we need to reject it after 5 seconds?

Reply:
I guess we could leave it as pending forever until resolve? I put it there to clean up the promise and give some sort of log for user if answers is not initialize.

Reply:
What are your thoughts on having the reject called if the Answers init fails, rather than rejecting after 5 seconds? I think that should be possible by adding a `.catch` to the `Answer.init()` call

Reply:
oh yeah, that works!